USAGE.md

```diff
@@ -1,5 +1,44 @@
 # Usage
 
-Applications which use `WSTS` will typically run both `Signer` and `Coordinator` state machines, in order to be able to handle all parts of the protocol.  Because of this, these state machines do not verify packets as they come in; this would lead to duplicate work, and require the state machines to have all keys in their configs, including knowing who is the active coordinator.  This is out of scope for the library.  Thus applications `must` verify packets before calling `process_message` on them.
+Applications which use `WSTS` will typically run both `Signer` and `Coordinator` state machines. To ensure flexibility, these state machines require the user to provide a random number generator (RNG) that implements the `RngCore` and `CryptoRng` traits.
+
+### Providing an RNG
+
+You can use the following RNG implementations:
+
```

Comment:
The text at the bottom referring to the audit needs to come immediately after the discussion of the application’s responsibilities

Reply:
Thanks for catching this. Fixed